tools/run_tests/performance/build_performance.sh

```diff
@@ -55,6 +55,9 @@ do
   "csharp")
     python tools/run_tests/run_tests.py -l "$language" -c "$CONFIG" --build_only -j 8 --compiler coreclr
     ;;
+  "node"|"node_purejs")
+    tools/run_tests/performance/build_performance_node.sh
```

Comment:
```
[CXX]     Compiling test/cpp/end2end/test_service_impl.cc
[CXX]     Compiling test/cpp/util/byte_buffer_proto_helper.cc
[CXX]     Compiling test/cpp/util/channel_trace_proto_helper.cc

2018-04-30 21:51:35,996 TIMEOUT: remote_host_build.grpc-kokoro-performance-client-8core [pid=6354, time=909.1sec]
```
It looks like it's timing out when trying to prepare the remote workers. I suspect this is simply because of the added overhead of building Node and not any actual issue. Is there anything that can be done here to speed up building Node?

We can also increase the timeout. [`build_timeout`](https://github.com/grpc/grpc/blob/master/tools/run_tests/run_performance_tests.py#L250) and [`local_build_timeout`](https://github.com/grpc/grpc/blob/master/tools/run_tests/run_performance_tests.py#L252) can be combined to be `build_timeout = 30 * 60` since it's timing out when building on remote hosts. This [reference](https://github.com/grpc/grpc/blob/master/tools/run_tests/run_performance_tests.py#L272) will also need to be updated.

Reply:
The comments in that file seem to indicate that the `local_build_timeout` needs to be longer than the `build_timeout`. So does it make more sense to increase both?

Reply:
The `local_build_timeout` didn't timeout on your build, but I'm okay with increasing both anyways.